src/lightning/MANIFEST.in

```diff
@@ -0,0 +1,12 @@
+include src/version.info
```

Comment:
This is the only file I'm very uncertain about. Could use a detailed review of this file. Not sure why we didn't already have one, maybe there's some kind of magic happening behind the scenes somewhere.

Reply:
This manifest shall generated as aggregation of all three others

Reply:
Gotcha, so if I delete this file, will it still add a py.typed to the wheel?

Reply:
Yes @adamjstewart